lib/task_helpers.rb

```diff
@@ -12,30 +12,40 @@ module ElasticsearchHelper
     #
     # Params:
     # +alias_name+:: (optional) The alias to point to the new index.
-    def self.rebuild_index(alias_name=nil)
+    # +batch_size+:: (optional) The number of elements to index at a time. Defaults to 100
```

Comment:
Great!  Can you add a period to the end of these two lines (i.e. "Defaults to 100.") when you squash and merge?  Thanks.